packages/workbox-core/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "workbox-core",
+  "version": "1.2.0",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
+  "description": "This module is used by a number of the other workboxjs.org modules to share common code.",
+  "repository": "googlechrome/workbox",
+  "bugs": "https://github.com/googlechrome/workbox/issues",
+  "homepage": "https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-core",
+  "keywords": [
+    "workbox",
+    "workboxjs",
+    "service worker",
+    "sw"
+  ],
+  "scripts": {
+    "pretest": "node ../../infra/build-test-bundle.js",
+    "test": "mocha './test/bundle-build/**/*.js'"
```

Comment:
I know that `gulp test` does a lot of setup—starting up our test server, running the multi-browser testing harness, etc.

Is it sufficient to just run `mocha` directly here rather than going through the gulp task? Or is this just meant to be a subset of the tests?